### DISCUSSION TITLE
Community update October 2023

### DISCUSSION THREAD

**Discussion by vlidholt** [serverpod/serverpod#1490] [Announcements]
Greetings from Serverpod! 🚀

Watch the video update here:
https://www.youtube.com/watch?v=_a6OZ_rDBEg

## Upcoming 1.2 version
We're thrilled to share some insights into the next Serverpod version. Our passionate team has been putting more hours into this update than all our previous versions combined! We're striving to offer you an experience that's more powerful and reliable than ever, setting a solid foundation for many future features.

## New features alert 🚨
- **Database ORM updates:** We're unveiling many new database features, including one-to-one, many-to-one, and many-to-many relations—all through a type-safe Dart API. We are also making database migrations a breeze with our new tooling.
- **VS Code plugin:** Attention all VS Code enthusiasts! We are developing a plugin to validate Serverpod model files with our brand-new language server. Plus, our enhanced analyzer will now provide even more precise error feedback.
- **Improved CLI tooling:** Expect clearer outputs and more precise error messages. We're refining the user experience, making it more intuitive than ever!

## Stability and tests
We've implemented over 1,000 tests to strengthen Serverpod's stability and resilience. This not only ensures a solid update for you but also makes it easier for contributors and faster for us to review pull requests.

## Join the team
We're on the hunt for talented Dart and Flutter developers and a developer evangelist! We're primarily recruiting in Stockholm, but outstanding candidates from anywhere can join our remote team. Interested? Explore opportunities on our [careers page](https://careers.serverpod.dev/).

## A very special thank you 🎁
For all those who've contributed to the upcoming version, we have a surprise in store! If you've added code or reported issues, please fill out our [contributors form](https://forms.gle/TecwXHftMW1R9R438) to ensure you don't miss out.

## In closing
Your support pushes us to make every new update better. While we're as eager as you are for the 1.2 version, we're committed to delivering nothing but the best. Stay tuned for more updates soon, and make sure you're following us on our social channels so you do not miss out!

Until next time, happy coding!

The Serverpod team

**Comment by ps60679666** [comment]
Is there specific date for this update (Serverpod 1.2)? 
Thank you.

**Reply by Isakdl** [reply]
We have an RC out now on pub dev! https://pub.dev/packages/serverpod/versions/1.2.0-rc.1
